resource/src/lib.rs

```diff
@@ -271,11 +269,10 @@ mod tests {
     use std::fs;
     use std::path::{Path, PathBuf};
 
-    fn mkdir() -> tempfile::TempDir {
-        tempfile::Builder::new()
-            .prefix("ckb_resource_test")
-            .tempdir()
-            .unwrap()
+    fn mkdir() -> PathBuf {
```

Comment:
Does `/tmp/ckb_resource_test` folder create successful on windows? I think we may keep the test code unchanged here, just move `tempfile` crate to `[dev-dependencies]` section.

Reply:
ok,  you are right, let's move tempfile crate into dev-dependencies and untouch test code.